session_0/fizz_buzz.rb

```diff
@@ -37,5 +37,23 @@
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
 def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
+  values=Array.new()
```

Comment:
Hello!

Your answer looks correct but can be improved further:
- Since we know the number of iterations to be made (exactly `n` iterations), try using a for loop instead of a while loop.
- See if [map function](https://www.rubyguides.com/2018/10/ruby-map-method/) can be used here.
- Because of the associativity rules, we avoid using `and`, `or` in ruby and instead use `&&`, `||` operators: https://rubystyle.guide/#no-and-or-or

Reply:
@abhishekkumar2718  Thank you for the suggestions.
I will try to improve the code